spec/openapi.yaml

```diff
@@ -508,7 +508,7 @@ paths:
           type: string
           enum: ["python"]
```

Comment:
Should not we rename it to `system` or `link` then?

Reply:
Good point, I'll find out